documentation/asciidoc/topics/ref_services.adoc

```diff
@@ -46,7 +47,25 @@ are not available by default.
 | `${cluster_name}-external`
 | `11222`
 | TCP
-| {brandname} endpoints
+| External access to {brandname} endpoints.
+
+|===
+
+.Cross-site service
+
+Allows {brandname} to back up data between clusters in different locations.
+
+[%header,cols=4*]
+|===
+| Service
+| Port
+| Protocol
+| Description
+
+| `${name}-sites`
+| `7900`
```

Comment:
@galderz is `7900` the correct port here? I got this from https://gist.github.com/galderz/7ece0b788a6fa6559906b85c592c81de#file-jgroups-relay-xml-L2